site/src/js/index.js

```diff
@@ -29,6 +55,8 @@ const renderSwag = () => {
             case 'Alphabetical, reversed': return a.name.toLowerCase() < b.name.toLowerCase() ? 1 : -1;
             case 'Difficulty':             return difficultyIndex(a.difficulty) > difficultyIndex(b.difficulty) ? 1 : -1;
             case 'Difficulty, reversed':   return difficultyIndex(a.difficulty) < difficultyIndex(b.difficulty) ? 1 : -1;
+            case 'Popular':                return claps[`devswag.io/${a.name}`] < claps[`devswag.io/${b.name}`] ? 1 : -1;
```

Comment:
You may want to move the `devswag.io` prefix to a constant string, to reduce repetition.